_data/Brian-Hou/Brian-Hou.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Brian Hou
+institution: University of Washington Bothell
+profile_pic: Brian-Hou.jpg 
+quote: "NYTimes 09/Apr/2020 With $2.3T Injection, Fed's Plan Far Exceeds 2008 Rescue."
```

Comment:
Remove double quotes from begin and end.

Reply:
Got it!